crates/geoblock/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "geoblock"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = []
+#full = ["metrics"]
```

Comment:
Initially, I thought about having one middleware for Tower, and one for Warp, hence the placeholder for features. Now since Warp has become so niche, I'm not sure it's worth having it in `utils`. Maybe it's easier to just put it directly in `relay`. WDYT?

Reply:
Yea, fuck `warp`, we should just migrate the `relay` already.